oauthproxy.go

```diff
@@ -1054,4 +1060,7 @@ func isAjax(req *http.Request) bool {
 func (p *OAuthProxy) errorJSON(rw http.ResponseWriter, code int) {
 	rw.Header().Set("Content-Type", applicationJSON)
 	rw.WriteHeader(code)
+	// we need to send some JSON response because we set the Content-Type to
+	// application/json
+	rw.Write([]byte("{}"))
```

Comment:
Perhaps we should include the status message in here? IE something like `"message": "Unauthorized"` for a 401 code. We have some stuff like this within the pagewrite package already, perhaps it's worth extending the pagewrite interface to write the JSON as well and then we can share that logic for getting the message

Reply:
The only area I'm torn is this adds JSON formatting to match a JSON schema the underlying backend API has. So I lean toward `{}` to not lead to any unintentional overlaps with formatting the backend might have (and frontent client might assume).